src/js/components/Tiles.js

```diff
@@ -161,16 +164,29 @@ export default class IndexTiles extends Component {
           </Tiles>
         );
 
+        let bulkOperationsContent;
+        let sectionContent = sectionTiles;
+
+        if (bulkOperationsComponent) {
+          bulkOperationsContent = <BulkOperations items={sectionItems} component={bulkOperationsComponent}/>;
+          sectionContent = (
+            <Box key={section.label} direction="row" pad={{between: 'small'}} responsive={false}>
+              {sectionTiles}
+              {bulkOperationsContent}
+            </Box>
+          );
+        }
+
         if (sections.length !== 0 || items.length !== 0) {
           // more than one section, add label
           sections.push(
             <div key={section.label} className={`${CLASS_ROOT}__section`}>
               <label>{section.label}</label>
```

Comment:
I would expect the actions to be in a header-isa container along with the section.label, as they go together semantically.